targets/targets.json

```diff
@@ -5091,6 +5091,45 @@
         "features": ["BLE"],
         "release_versions": ["2", "5"]
     },
+    "ARM_MUSCA_A1": {
+        "inherits": ["Target"],
+        "core": "Cortex-M33-NS",
+        "device_has": ["INTERRUPTIN", "LPTICKER", "SERIAL", "SLEEP", "USTICKER"],
+        "macros": [
+            "MBED_TZ_DEFAULT_ACCESS=1",
+            "__STARTUP_CLEAR_BSS",
+            "MBED_FAULT_HANDLER_DISABLED",
+            "CMSIS_NVIC_VIRTUAL",
+            "LPTICKER_DELAY_TICKS=1",
+            "MBED_MPU_CUSTOM"
+        ],
+        "extra_labels": ["ARM_SSG", "MUSCA_A1", "PSA", "TFM"],
+        "supported_toolchains": ["ARMC6", "GCC_ARM"],
+        "default_toolchain": "ARMC6",
```

Comment:
I don't believe `ARMC6` is a valid setting for this field. It's pretty confusing, but due to legacy reasons this controls whether a target uses the "microlib" or not when being built in the online compiler (which only supports ARM compiler 5). I would recommend just dropping the line.
```suggestion
```